coffea/lookup_tools/txt_converters.py

```diff
@@ -183,8 +183,8 @@ def _build_standard_jme_lookup(
                     theBins = np.union1d(binMins, binMaxs[-1:])
                 allBins = np.append(allBins, theBins)
                 counts = np.append(counts, theBins.size)
-            bins[layout[i + offset_name]] = awkward.JaggedArray.fromcounts(
-                counts, allBins
+            bins[layout[i + offset_name]] = ak.unflatten(
```

Comment:
Did you cross check the minimum awkward version for `unflatten` in setup.py?

Reply:
1.0.1 is good and in setup.py

Reply:
This raises a question if we want awkward release candidates as pins.
Right now I have to use uproot4 rc because otherwise it reads histograms wrong.

Delaying this to rc2.